Source/debug.cpp

```diff
@@ -565,7 +565,7 @@ std::string DebugCmdLevelUp(const std::string_view parameter)
 	Player &myPlayer = *MyPlayer;
 	std::string cmdLabel = "[givelvl] ";
 
-	const int levels = ParseInt<int>(parameter, /*min=*/1).value_or(1);
+	const int levels = std::min<int>(ParseInt<int>(parameter, /*min=*/1).value_or(1), GetMaximumCharacterLevel());

```

Comment:
Actually, I just noticed that `ParseInt()` has a max parameter.
```suggestion
	const int levels = ParseInt<int>(parameter, /*min=*/1, /*max=*/GetMaximumCharacterLevel()).value_or(1);
```

Reply:
Oh, never mind. We don't want that, since it would cause parsing to fail, changing the number to 1 instead of clamping it to 50.

Reply:
Done :)